src/main/java/uk/gov/hmcts/reform/ccd/data/entity/CaseLinkEntity.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.ccd.data.entity;
```

Comment:
I have written the equivalent of this in datastore - it has been mentioned before that perhaps we have a common library of domain objects. (and in this case some entity objects) ? Its outside the scope of this review I know but worth mentioning ?